modules/ocf_www/manifests/logging.pp

```diff
@@ -7,6 +7,29 @@
     require => Package['nfs-kernel-server'],
   }
 
+  # log outbound requests
+  firewall_multi {
+    '101 log outbound request on death (v4)':
+        provider    => 'iptables',
+        chain       => 'PUPPET-OUTPUT',
+        outiface    => '! lo',
+        jump        => 'LOG',
+        destination => '! 169.229.226.0/24',
```

Comment:
Here and below: is it possible to get this netblock from Hiera or the `ocf::networking` class, instead of duplicating it?

Reply:
I don't think we have this stored and while it's in the networking class I'm not sure how to get it out of there